rasa/core/policies/embedding_policy.py

```diff
@@ -270,7 +270,9 @@ def _create_session_data(
             label_ids = None
             Y = None
 
-        return train_utils.SessionData(X=data_X, Y=Y, label_ids=label_ids)
+        return train_utils.SessionData(
+            X={"X": data_X}, Y={"Y": Y}, labels={"labels": label_ids}
```

Comment:
it seems to me the keys are arbitrary and for each algorithm to create. Then, I think it makes sense to use more natural keys
```suggestion
            X={"dial": data_X}, Y={"bot": Y}, labels={"actions": label_ids}
```